### DISCUSSION TITLE
Trying to get openchrom to run in eclipse

### DISCUSSION THREAD

**Discussion by ScottyTarvan** [OpenChrom/openchrom#476] [Q&A]
Hello, I have been trying to get openchrom running in Eclipse by following your DEV guidelines with no success. Latest issue I have is that it "Unable to locate installable unit net.openchrom.rcp.compilation.community.converter.feature.feature.group 0.0.0".
I'm really looking for the code that opens and converts an Agilent Data.Ms file - can you point me in right direction please. 
Tx.

**Comment by Mailaender** [comment]
Sorry, the vendor formats are mostly closed source as reverse engineering them involved a lot of work. Running in Eclipse may not always work. Try https://github.com/OpenChrom/openchrom/wiki/Build in the command line instead.
